src/core/dom/layout/viewport-observer.js

```diff
@@ -106,8 +106,7 @@ function ioCallback(entries) {
       continue;
     }
     for (let k = 0; k < callbacks.length; k++) {
-      const callback = callbacks[k];
-      callback(entry);
+      callbacks[k].?();
```

Comment:
My bad, if its in the array it should always exist right?
```suggestion
      callbacks[k]();
```

Reply:
Yes, `callbacks` is an array.

Reply:
This actually raises an interesting question -- do we ever want an observed element to be able to deregister a specific callback without voiding all others attributed to that element?

Reply:
Does it not handle that properly in the current PR?

Reply:
Not sure, does it? Where would I find it?